commands/askSilverwolfAI.ts

```
@@ -25,69 +25,103 @@ class AskSilverwolfAI extends Command {
+      if (toolCalls?.length) {
+        for (const tc of toolCalls) {
+          await this.client.db.aiChat.addHistory(aiSession.sessionId, 'tool', JSON.stringify(tc));
+        }
+      }
+      if (text) {
+        await this.client.db.aiChat.addHistory(aiSession.sessionId, aiRole, text);
+      }
+
+      if (!hadRawHistory && text) {
+        this.client.db.aiChat.getHistory(aiSession.sessionId, 100)
+          .then((history: { role: string; message: string }[]) => generateTitleForHistory(history))
```

ESLint

[error] 'history' is already declared in the upper scope on line 52 column 31.

(no-shadow)